go.mod

```diff
@@ -0,0 +1,24 @@
+module k8s.io/release
+
+go 1.12
+
+require (
+	cloud.google.com/go v0.41.0 // indirect
+	github.com/blang/semver v3.5.1+incompatible
+	github.com/go-kit/kit v0.9.0
+	github.com/go-logfmt/logfmt v0.4.0 // indirect
```

Comment:
I think we can remove most of these indirect dependencies since they are not needed. Unfortunately `go mod tidy` doesn't to the job very well here.

Reply:
@saschagrunert -- is that something we can take as a follow-up?

Reply:
Sure, let’s do that.